rules/linux/discovery_pspy_process_monitoring_detected.toml

```
@@ -0,0 +1,71 @@
+[metadata]
+creation_date = "2023/07/20"
+integration = ["auditd_manager"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/07/20"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule leverages auditd to monitor for processes scanning different processes within the /proc directory using the 
+openat syscall. This is a strong indication for the usage of the pspy utility. Attackers may leverage the pspy process 
+monitoring utility to monitor system processes without requiring root permissions, in order to find potential privilege 
+escalation vectors.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-auditd_manager.auditd-*"]
```

```suggestion
index = ["logs-auditd_manager.auditd-*"]
```
As it filters on `event.dataset == "auditd_manager.auditd"`